charts/gateway-addons-helm/values.yaml

```diff
@@ -1,3 +1,6 @@
+dashboards:
```

Comment:
can we use `dashboard` here ? because the object is not an array/map type

Reply:
although the generated object is a single ConfigMap, but it creates multiple grafana dashboards, doesn't it? so I think using the plural form is more accurate.

Reply:
okay, it's been done.